src/backend/rendering/chtmlexportrendering.cpp

```diff
@@ -234,7 +222,8 @@ QString CHTMLExportRendering::finishText(QString const & text,
     \fn CHTMLExportRendering::entryLink( KeyTreeItem& item )
  */
 QString CHTMLExportRendering::entryLink(KeyTreeItem const & item,
-                                        CSwordModuleInfo const *)
-{ return item.key(); }
+                                        const CSwordKey * key) {
```

Comment:
Please consider removing the name `key` or adding `Q_UNUSED(key);` as this might otherwise probably cause a compiler warning.

Reply:
I would like to remove that parameter at all. And maybe remove some another methods/constructors that do not make much sense.